specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-03-01-preview/subscription.json

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
Rename file to "subscriptions.json" to be consistent with the (existing) stable version.